src/instruction.rs

```diff
@@ -94,3 +103,14 @@ pub fn noop() -> Instruction {
     data: PythClientInstruction::Noop.try_to_vec().unwrap(),
   }
 }
+
+// Returns ok if price is not stale
```

Comment:
comment is wrong now that you have expected_price_status

Reply:
Fixed.